pages/atom.html

```diff
@@ -0,0 +1,32 @@
+---
+path: atom.xml
+data:
+  hidden: true
+---
+<?xml version="1.0"?>
+{% assign section = 'issues/index.html' | sectionOf %}
+{% assign children = section.pages | sortedByWeight  %}
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <title>This Week in Dart</title>
+  <author><name>Parker Lougheed</name></author>
+  <link type="application/atom+xml" rel="self" href="{{ '' | pageUrl }}"/>
+  <id>{{ '' | pageUrl }}</id>
+  <contributor><name>This Week in Dart contributors</name></contributor>
+  <rights type="html">
+    Made available by Parker Lougheed and Contributors under the &lt;a href=&quot;http://creativecommons.org/licenses/by/4.0/&quot;&gt;Creative Commons Attribution 4.0 License&lt;/a&gt;
```

Comment:
Can this link be `https` or is this a limitation of the atom format?

```suggestion
    Made available by Parker Lougheed and Contributors under the &lt;a href=&quot;https://creativecommons.org/licenses/by/4.0/&quot;&gt;Creative Commons Attribution 4.0 License&lt;/a&gt;
```

Reply:
I've copied the URL from the [LICENSE](https://github.com/parlough/thisweekindart/blob/main/LICENSE) file in this repo without taking a deeper look, we should probably change both :D

Reply:
Oops! I didn't realize, thanks for pointing that out. I've adjusted the other CC links.